web/css/source/variables/overrides/_typography.less

```diff
@@ -0,0 +1,3 @@
+// Typography
+@text__color: blue;
```

Comment:
try to use your own solutions, not mine from the video. and try to use some good colors from the FE/designer's point of view. I mean, blue color is not nice for a simple text if you know what I mean 😏